gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -282,42 +284,77 @@ mehrbedarf_anteil:
     max: 0.6
     reference: G. v. 24.12.2003 BGBl. I S. 2954.
 
-vermögensfreibetrag:
+
+vermögensfreibetrag_obergrenze:
   name:
-    de_1: Vermögensgrundfreibetrag je Lebensjahr, nach Geburtsjahr, erst ab 2008 unterschiedlich
-    de_2: Vermögensfreibetrag für Erstausstattung
-    de_3: Vermögensgrundfreibetrag je Kind
-    en_1: Basic allowance per year of life, sorted by year of birth, starting 2008 different
-    en_2: Allowance for furniture
-    en_3: Allowance per child
+    de: Maximaler Vermögensgrundfreibetrag je Lebensjahr. Ab 2008 nach Geburtsjahr unterschiedlich
```

Comment:
Explanations like "Ab 2008 nach Geburtsjahr unterschiedlich" should be part of `description` rather than `name`.

Reply:
Moved to description!